listenbrainz/webserver/views/user.py

```diff
@@ -252,3 +278,18 @@ def delete_listens_history(musicbrainz_id):
     _ts.reset_listen_count(user.musicbrainz_id)
     db_user.reset_latest_import(user.musicbrainz_id)
     db_stats.delete_user_stats(user.id)
+
+
+def delete_listen(musicbrainz_id: str, recording_msid: str):
```

Comment:
You have two delete_listen functions here. And this one does very little, do we really need it?

Reply:
Actually seems unnecessary. So nuked it